src/content/tutorial/react-step-2/react-step-2.mdx

```diff
@@ -0,0 +1,641 @@
+      <div className="bx--col-lg-8">1/2</div>
+      <div className="bx--col-lg-8">1/2</div>
+    </div>
+    <div className="bx--row landing-page__r3">
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+    </div>
+    <div className="bx--row landing-page__r4">
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-8">1/2</div>
+    </div>
+    <div className="bx--row landing-page__r5">
+      <div className="bx--offset-lg-4 bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-8">1/2</div>
+    </div>
+    <div className="bx--row landing-page__r6">
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+      <div className="bx--col-lg-4">1/4</div>
+    </div>
+  </div>
+);
+```
+
+We added a class of `bx--grid--full-width` to the grid container since our rows need to expand the whole page without any margins. We also added some custom classes like `landing-page`, `landing-page__banner`, `landing-page__r2`, etc, which we will use later.
+
+Another important thing to notice is that in the 5th row `landing-page__r5`, instead of including two `bx--col-lg-4` `div`'s, I only included one, and gave it an additional class of `bx--offset-lg-4`. This is because we won't actually have any content in the first quarter column, so instead of having an empty column, we can just offset the second column by a quarter using the additional class.
+
+## Add landing page content
+
+We'll start adding HTML elements and components by row.
+
+#### First row
+
+In our first row we'll need a `<BreadCrumb/>` component. First, let's import the components we need. Because we'll be importing several components for this page, we'll import them directly from the `carbon-components-react` package instead of the direct path for each one.
+
+```javascript
+import { Breadcrumb, BreadcrumbItem } from 'carbon-components-react';
+```
+
+We can now add our component to the first row, along with a header, like so:
+
+```html
+<div className="bx--row landing-page__banner">
+  <Breadcrumb noTrailingSlash>
+    <BreadcrumbItem>
+      <a href="/">Getting started</a>
+    </BreadcrumbItem>
+  </Breadcrumb>
+  <h1>Design &amp; build with Carbon</h1>
+</div>
+```
+
+You may notice that the styles look off. Don't worry, we'll fix these later.
+
+#### Second row
+
+In our second row we'll need `<Tabs/>` and `<Button/>` components. We'll add those to our import at the top of the file.
+
+```javascript
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+```
+
+Before we can render the tabs, we need to add some props for the component after the import.
+
+```javascript
+const props = {
+  tabs: () => ({
+    className: 'some-class',
```

Comment:
I'm imagining that you ported the code from our Storybook code. Please note that the props being function there is the nature of Storybook knobs. For props in applications props here can be simple key-map values.

Update: Or you can directly put the props to below React rendering code.

Reply:
Thanks for the explanation! I updated the code! 👍